rakelib/labels.rake

```diff
@@ -0,0 +1,80 @@
+require 'octokit'
+
+namespace :labels do
```

Comment:
Thought about it, one syncs files though, the other syncs github repo stuff. Can we repeat the namespace? 🤔